x-pack/plugins/apm/common/dependencies.ts

```diff
@@ -13,13 +13,15 @@ import {
 } from './es_fields/apm';
 import { environmentQuery } from './utils/environment_query';
 
-export const kueryBarPlaceholder = i18n.translate(
-  'xpack.apm.dependencies.kueryBarPlaceholder',
+export const unifiedSearchBarPlaceholder = i18n.translate(
+  'xpack.apm.dependencies.unifiedSearchBarPlaceholder',
   {
     defaultMessage: `Search dependency metrics (e.g. span.destination.service.resource:elasticsearch)`,
   }
 );
 
+// TODO: This function must be deleted in favour of Unified Search.
+//  Since we are keeping the KueryBar code for now, will delete this once we get rid of that as well
 export const getKueryBarBoolFilter = ({
```

Comment:
it seems the function is not used anywhere? Why do we need to keep it?

Reply:
I added a TODO to delete this function only when we are sure we don't need any of the old KueryBar code in the future.
But looks like we will need this function now once we have this ticket solved - https://github.com/elastic/kibana/issues/154437

Reply:
I think we can simplify the existing behaviour so we just filter by `service.name` and possibly `service.environment`. We don't have to filter by `processor.event`.